praetorian_cli/sdk/entities/definitions.py

```
@@ -29,18 +29,26 @@ def add(self, local_filepath, definition_name=None):
         :param download_directory: The directory to save the downloaded file (defaults to current working directory)
         :type download_directory: str
+        :param global_: If True, fetch from global definitions instead of user-specific
+        :type global_: bool
         :return: The local file path where the definition was saved
         :rtype: str
         """
-        content = self.api.files.get_utf8(f'definitions/{definition_name}')
+        if global_:
+            try:
+                content = self.api.download(f'definitions/{definition_name}', global_=True).decode('utf-8')
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)